model/riscv_platform.sail

```diff
@@ -319,7 +319,7 @@ function clint_store(addr, width, data) = {
 val tick_clock : unit -> unit effect {rreg, wreg}
 function tick_clock() = {
   if   mcountinhibit.CY() == 0b0
```

Comment:
Why not fold the CSR checks into the functions?

Reply:
Done